doc/Data Plane Code Generation/Sample.md

```diff
@@ -0,0 +1,28 @@
+Samples are where developers could start to learn a library. You can generate samples scaffold by running [quick start tool](AzureSDKCodeGeneration_DataPlane_Quickstart.md#create-starter-package), or start from scratch. Both of them need to generate the complete set of samples which contain the following key features complying with the structure:
```

Comment:
> start from scratch.

I don't think we want people to start from scratch -- partners creating a DPG should use the starter script.  The samples generated by the starter script should match the structure of the samples described in this document.